contribs/gmf/apps/desktop/js/controller.js

```diff
@@ -8,15 +8,30 @@ goog.provide('app.desktop.Controller');
 
 goog.require('app');
 goog.require('gmf.controllers.AbstractDesktopController');
+goog.require('gmf.import.module');
 goog.require('ngeo.proj.EPSG2056');
 goog.require('ngeo.proj.EPSG21781');
 goog.require('ol');
 
 app.desktop.module = angular.module('AppDesktop', [
   app.module.name,
   gmf.controllers.AbstractDesktopController.module.name,
+  gmf.import.module.name,
 ]);
 
+app.desktop.module.value('gmfExternalOGCServers', [{
```

Comment:
What's those gmfExternalOGCServers ?

Reply:
It is the prefilled value, I just took what was done in desktop_alt.